### PR TITLE
388 add footer

### PR DESCRIPTION
**Pull request by ubbabeck**
# This PR references issue #388 

I am fairly new to flutter and UI, please let me know if I am not following best practice.

## About commits
8e73667 Expand preferences menu by default.
3b0394a add AppMode to UserProfileSettings, we only display balances until apps are supported
37fe6ed & 19909d9 Creates a new footer method and widget
064e6ce & 2b6a53e UI fixup

63b7494 display horizontally, update font

## UI

greenlight-android
greenlight-iphone

### CODE REVIEW

**Review by ubbabeck — COMMENTED** [review]
Thank you 🙏 look much better.

**Comment by kingonly** [comment]
@ubbabeck looks good! Why is there a place in the footer in the light theme? The dark theme looks right.

**Comment by ubbabeck** [comment]
@kingonly thank you! The light themed photo is on Android and the dark one is on iOS. Not sure why this happens on Android, can see if I can fix it.

**Comment by erdemyerebasmaz** [comment]
> @ubbabeck looks good! Why is there a place in the footer in the light theme? The dark theme looks right.

The system bottom navigation bar height is taken into consideration now and divider is aligned with bottom actions bar.
_Ignore the footer content_
|iOS|Android|
|---|---|
|File|Screenshot_20230127_153505_com cBreez client|

**Comment by kingonly** [comment]
I'll ask the designer to create an image.

**Comment by ubbabeck** [comment]
Received the image from designers, what do you think @kingonly and @erdemyerebasmaz ?
light mode is android, dark mode is iOS
pow_green_sdk
powered_by_breez-sdk_greenlight

**Comment by erdemyerebasmaz** [comment]
@ubbabeck Dividers alignment w/ bottom action bar seems broken and padding is a bit off. I'll send a footer image trimmed of it's surrounding transparent pixels maybe that'll help.

**Comment by ubbabeck** [comment]
fix

File: lib/routes/home/widgets/drawer/breez_navigation_drawer.dart [hunk 1]
```diff
@@ -171,6 +180,38 @@ class BreezNavigationDrawer extends StatelessWidget {
...
+          _kBreezBottomSheetHeight + MediaQuery.of(context).viewPadding.bottom,
+      child: Column(
+        children: [
+          const Divider(),
+          Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            crossAxisAlignment: CrossAxisAlignment.center,
+            children: [
+              Image.asset(
+                "src/images/breez-sdk_greelight_footer.png",
```
Comment (erdemyerebasmaz) [inline lib/routes/home/widgets/drawer/breez_navigation_drawer.dart:202]: Let's rename the file to `drawer-footer`.

**Review by erdemyerebasmaz — APPROVED** [review]
Looks good!
Approved with two minor comments.

**Pull request merged**